presto-main/src/main/java/com/facebook/presto/sql/relational/Expressions.java

```diff
@@ -64,6 +65,16 @@ public static InputReferenceExpression field(int field, Type type)
         return new InputReferenceExpression(field, type);
     }
 
+    public static VariableReferenceExpression variable(String name, Type type)
```

Comment:
If you really need this (which is good), extract it as a separate commit and change all places using `VariableReferenceExpression::new`. There are a lot.